examples/convert_ieeg_to_bids.py

```diff
@@ -115,13 +115,12 @@
 # - `background on FreeSurfer`_
 # - `MNE-Python coordinate frames`_
 #
-# Currently, MNE-Python supports the ``mni_tal`` and ``mri`` coordinate frames,
+# MNE-Python supports uses ``mni_tal`` and ``mri`` coordinate frames,
 # corresponding to the ``fsaverage`` and ``ACPC`` (for an ACPC-aligned T1) BIDS
 # coordinate systems respectively. All other coordinate coordinate frames in
-# MNE-Python if written with :func:`mne_bids.write_raw_bids` are written with
-# coordinate system ``'Other'``. Note, then we suggest using
-# :func:`mne_bids.update_sidecar_json` to update the sidecar
-# ``*_coordsystem.json`` file to add additional information.
+# MNE-Python, if written with :func:`mne_bids.write_raw_bids`, must have
+# an :attr:`mne_bids.BIDSPath.space` specified, and will be read in with
+# the montage channel locations set to the coordinate frame 'unknown'.
```

Comment:
Is this true? Or perhaps it can just be made a little bit more clear.

Maybe it would also help to provide a direct link to the specification for that matter?

https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/04-intracranial-electroencephalography.html#electrode-description-_electrodestsv

Reply:
If it's not in `MNE_TO_BIDS_FRAMES` and there is no `BIDSPath.space` then it won't be written. It could default to 'Other' that's a -1 from me, I don't think `Other` should be in BIDS.

Reply:
You get

```
warn("Coordinate frame could not be inferred from the raw object "
                 "and the BIDSPath.space was none, skipping the writing of "
                 "channel positions")
```

Reply:
It's linked at the top, we could add it there, but the whole narrative is trying to get them not to save data in a template coordinate frame if possible so handing a list of template coordinate frames that close to the beginning of the example seems counterproductive